### ISSUE TITLE
Unreadable about menu in Linux

### ISSUE THREAD

**Issue by hadisfr** [getinsomnia/insomnia#2614]
**Describe the bug**
About menu in Linux Ubuntu is totally filled with unreadable square-like characters.

**To Reproduce**
1. Go to 'Help' menu
2. Click on 'About'
3. See error

**Expected behavior**
Seeing readable contents.

**Screenshots**
![image](https://user-images.githubusercontent.com/24562441/93184541-9d330680-f751-11ea-8561-6ae00c82f310.png)

**Desktop**
 - OS: Ubuntu 20.04.1 LTS
 - Installation Method: `snap` (latest/stable channel)
 - App Version 2020.4.0 2020-09-14

**Additional context**
When running from terminal, only the following warning appears after opening about menu, which seems unimportant:
```console
$ insomnia
...
Gtk-Message: 12:45:37.468: GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by develohpanda** [comment]
Thanks for the report! I wonder if it is related to missing fonts. Did it work for you in 2020.3.3 (the previous stable version)?

**Comment by hadisfr** [comment]
Unfortunately, I cannot test the previous version. After reverting to 2020.3.3 using `snap`  I get some no such file or directory related to `appmenu-gtk-module` (Seems `snap revert` bug or incomplete revert or sth).

**Comment by hadisfr** [comment]
But also a totally white screen appears whenever I click _Preferences_ in 2020.4.0. t may be related to this issue or not.

Logs in `--enable-logging` modes:
```
[19955:0916/015916.601753:INFO:CONSOLE(31635)] "[ga] Event Modals, Show, SettingsModal", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (31635)
Fontconfig warning: FcPattern object width does not accept value [75 100)
```

**Comment by develohpanda** [comment]
Just looked at the PR that introduced it (https://github.com/Kong/insomnia/pull/2505) and it contains a screenshot from Linux working correctly on Ubunutu 20.04.1. 🤔 INS-124.

**Comment by hadisfr** [comment]
Interesting… 🤔  

My config is (using the left button which seems to be copy):
```
Version: Insomnia Core 2020.4.0
Release date: 10/09/2020
OS: Linux x64 5.4.0-47-generic
Electron: 9.1.1
Node: 12.14.1
V8: 8.3.110.13-electron.0
Architecture: x64
node-libcurl: libcurl/7.69.1 OpenSSL/1.1.1g zlib/1.2.11 brotli/1.0.7 libssh2/1.9.0 nghttp2/1.41.0
```
So it seems to be just font problem, as you said. But I expected a warning (in terminal) for missing font, at least in debug mode. 🤔 
My full log:
```console
$ insomnia --enable-logging
Error org.freedesktop.DBus.Error.Failed: cannot set "default-url-scheme-handler" setting to invalid value "insomnia"
Gtk-Message: 13:10:30.694: Failed to load module "appmenu-gtk-module"
[8558:0916/131030.766220:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[fix] Running database repairs
[db] Initialized DB at /home/hadi/snap/insomnia/102/.config/Insomnia/insomnia.$TYPE.db
Init responses DB
[localstorage] Initialized at /home/hadi/snap/insomnia/102/.config/Insomnia/localStorage
[main] Loading file:///snap/insomnia/102/resources/app.asar/renderer.html
[ga] Non-interactive General, Launched, 2020.4.0
[ga] Queued event [
  { name: 't', value: 'event' },
  { name: 'ec', value: 'General' },
  { name: 'ea', value: 'Launched' },
  { name: 'ni', value: '1' },
  { name: 'el', value: '2020.4.0' }
]
[updater] Updater not running platform=linux dev=false
[8455:0916/131032.813694:INFO:CONSOLE(1214)] "[db] Initialized DB client", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (1214)
(node:8585) Electron: Loading non-context-aware native module in renderer: '/run/user/1000/snap.insomnia/.org.chromium.Chromium.OsKEUX'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:8585) Electron: Loading non-context-aware native module in renderer: '/run/user/1000/snap.insomnia/.org.chromium.Chromium.VTIjdY'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:8585) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
[8455:0916/131033.427580:INFO:CONSOLE(194290)] "[FileSystemDriver] Initialized in "/home/hadi/snap/insomnia/102/.config/Insomnia/version-control"", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (194290)
[8455:0916/131033.453656:INFO:CONSOLE(31635)] "[ga] Event Activity, Change, insomnia", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (31635)
[8455:0916/131033.453708:INFO:CONSOLE(31662)] "[ga] Page /insomnia", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (31662)
[8455:0916/131034.459063:INFO:CONSOLE(175771)] "[sync] Initialized", source: file:///snap/insomnia/102/resources/app.asar/bundle.js (175771)
Gtk-Message: 13:11:18.435: GtkDialog mapped without a transient parent. This is discouraged.
```

Is there any way I can test other versions without installation? _.zip_ artifact of release seems to have macOS version only.

**Comment by develohpanda** [comment]
Very strange

> Is there any way I can test other versions without installation? .zip artifact of release seems to have macOS version only.

Unfortunately there is no portable version, so the only way is to install it. Older versions should be under releases if you filter to `core@2020.` or `designer@2020.`.

When I get a change I will try this in a fresh VM running your version of Ubuntu. That might help to narrow down whether this is scoped to your installation, or to do with Ubuntu itself.

With the Electron upgrade introduced in 2020.4.0, there was an update to how the `font-manager` module was used, so it _could_ be related, but also possibly unrelated.

**Comment by miguelbaldi** [comment]
Same thing for me, on Fedora 32.
I've tested the AppImage version, and seems to work fine: https://updates.insomnia.rest/downloads/linux/latest
Problem only occurs on Snap version.

**Comment by miguelbaldi** [comment]
Log runnig with AppImage (works fine):
```
[fix] Running database repairs
[db] Initialized DB at /home/miguel/.config/Insomnia/insomnia.$TYPE.db
Init responses DB
[localstorage] Initialized at /home/miguel/.config/Insomnia/localStorage
[main] Loading file:///tmp/.mount_Insomn1GpvxB/resources/app.asar/renderer.html
[ga] Non-interactive General, First Launch, 2020.4.0
[ga] Queued event [
  { name: 't', value: 'event' },
  { name: 'ec', value: 'General' },
  { name: 'ea', value: 'First Launch' },
  { name: 'ni', value: '1' },
  { name: 'el', value: '2020.4.0' }
]
[updater] Updater not running platform=linux dev=false
(node:37028) Electron: Loading non-context-aware native module in renderer: '/tmp/.org.chromium.Chromium.UPLpr7'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:37028) Electron: Loading non-context-aware native module in renderer: '/tmp/.org.chromium.Chromium.HMDGy4'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:37028) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

Log running with Snap (does not work):
```
WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
Error org.freedesktop.DBus.Error.Failed: cannot use setting "default-url-scheme-handler": not allowed
/usr/share/libdrm/amdgpu.ids: No such file or directory
[fix] Running database repairs
[db] Initialized DB at /home/miguel/snap/insomnia/102/.config/Insomnia/insomnia.$TYPE.db
Init responses DB
[localstorage] Initialized at /home/miguel/snap/insomnia/102/.config/Insomnia/localStorage
[main] Loading file:///snap/insomnia/102/resources/app.asar/renderer.html
[ga] Non-interactive General, Launched, 2020.4.0
[ga] Queued event [
  { name: 't', value: 'event' },
  { name: 'ec', value: 'General' },
  { name: 'ea', value: 'Launched' },
  { name: 'ni', value: '1' },
  { name: 'el', value: '2020.4.0' }
]
[updater] Updater not running platform=linux dev=false
(node:37713) Electron: Loading non-context-aware native module in renderer: '/run/user/1000/snap.insomnia/.org.chromium.Chromium.cEsOYU'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:37713) Electron: Loading non-context-aware native module in renderer: '/run/user/1000/snap.insomnia/.org.chromium.Chromium.BqSAI7'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:37713) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

**Comment by sonicyeti** [comment]
I'm unable to reproduce this on Ubuntu 20.04.1 LTS running snap installed Insomnia Designer 2020.4.1.

![Image 2020-09-25 at 11 21 23 AM](https://user-images.githubusercontent.com/52717970/94285170-5a2b1d00-ff21-11ea-90b4-3b780512eb33.png)

**Comment by sonicyeti** [comment]
I'm also unable to reproduce this on Ubuntu 20.04.1 LTS running snap installed Insomnia Designer 2020.4

![Image 2020-09-25 at 12 32 59 PM](https://user-images.githubusercontent.com/52717970/94292798-6916cd00-ff2b-11ea-8c43-3ae8c2935a92.png)

**Comment by hadisfr** [comment]
@sonicyeti Could you check insomnia client, too, please?

**Comment by sonicyeti** [comment]
> @sonicyeti Could you check insomnia client, too, please?

Will do!

**Comment by sgohl** [comment]
edit: this happens with the snap package. the AppImage does work

I have this issue on Fedora 32 (xfce) with the following logs message:
```
[28210:0928/144128.350050:WARNING:browser_main_loop.cc(273)] GLib: getpwuid_r(): failed due to unknown user id (1010)
Gtk-Message: 14:41:28.464: GtkDialog mapped without a transient parent. This is discouraged.
```

That message is apparently caused by the fact that my user is not a classic system passwd/shadow user, but SSSD (Active Directory) bound. Insomnia seems to rely on a local system user id

**Comment by develohpanda** [comment]
> That message is apparently caused by the fact that my user is not a classic system passwd/shadow user, but SSSD (Active Directory) bound. Insomnia seems to rely on a local system user id

Ooh thanks for this, it might help narrow down what's causing it... @hadisfr and @miguelbaldi, is that the same case with your accounts as well?

**Comment by hadisfr** [comment]
> Ooh thanks for this, it might help narrow down what's causing it... @hadisfr and @miguelbaldi, is that the same case with your accounts as well?

Hmm interesting. It might be a case in snap version, too. But I would expect a related log as the one is in https://github.com/Kong/insomnia/issues/2614#issuecomment-699984493 to appear in my logs, if it's the case. I could not find such log in my full logs(https://github.com/Kong/insomnia/issues/2614#issuecomment-693265008). 🤔

**Comment by sonicyeti** [comment]
> @sonicyeti Could you check insomnia client, too, please?

Here is what I'm seeing w/ Core on the same VM.

![ubuntu-insomnia-core](https://user-images.githubusercontent.com/52717970/94561977-bb5d3400-0232-11eb-8621-11061e258951.png)

**Comment by hadisfr** [comment]
The problem seems similar to https://github.com/electron-userland/electron-builder/issues/5217.

**Comment by hadisfr** [comment]
OK. I found something.
It seems to be related to `fontconfig`'s bug (See https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484/18).
The workaround https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484/60 fixed the problem for me:
```bash
sudo rm /var/cache/fontconfig/*
rm ~/.cache/fontconfig/*
fc-cache -r
```
![image](https://user-images.githubusercontent.com/24562441/94565750-7529c400-0276-11eb-8446-9bd37fa64d02.png)
![image](https://user-images.githubusercontent.com/24562441/94565938-a99d8000-0276-11eb-9efa-0ba749776352.png)

@sgohl Can you test this please?
I think @sonicyeti used a new VM without any old font cache, so no problem was spotted.

**Comment by sgohl** [comment]
> @sgohl Can you test this please?

I did that before I commented here. Does not work for me. I also copied over all fonts from a ubuntu installation and rerun `sudo fc-cache --really-force` 

I think my problem with the user from sssd is like on-top of the base problem here

**Comment by sonicyeti** [comment]
My oldest Ubuntu VM is only a few months old, tried it though anyway and the issue isn't present.

**Comment by nijikokun** [comment]
@sgohl any help would be appreciated here, we cannot reproduce on the team and it looks like our current best solution for the issue is comment posted by @hadisfr which points to it being an underlying snap issue.

**Comment by sgohl** [comment]
> we cannot reproduce on the team

it is reproducable if you just install a fresh fedora xfce spin (not sure if classic gnome has same problem), install snapd and then insomnia. 
If you tell me I should test something particular, I will help happily

**Comment by Tony308** [comment]
I think it's a local issue **and** snap because I've got two separate apps installed from snaps;  which is insomnia designer and I've got the same issue with the fonts loading squares. Also have issue with #2707 which I presume is related to the fonts. Installation from other package managers seem fine.

**Comment by Tony308** [comment]
> I think it's a local issue **and** snap because I've got two separate apps installed from snaps; which is insomnia designer and I've got the same issue with the fonts loading squares. Also have issue with #2707 which I presume is related to the fonts. Installation from other package managers seem fine.

I uninstalled insomnia-designer from snap and used the AppImage instead. Works completely fine now. 

![image](https://user-images.githubusercontent.com/40594459/102914045-9fef7700-4477-11eb-9a4a-0a3b6de87f4a.png)

**Comment by supanadit** [comment]
![image](https://user-images.githubusercontent.com/33617149/152808625-56079753-4dd7-438e-8bd4-1e91a3e98eab.png)

Can someone explain this ? its only happen when istalling using snap, when i used Appimage has no issue at all

OS : Arch Linux 64 Bit
Gnome Version : 41.3
Installation Media : Snap
Hardware Model :  MSI GL65 Leopard 10SDK
Processor : Intel I7-10750H
Graphics : Nvidia GTX 1660 TI

**Comment by dimitropoulos** [comment]
did any of the advice in this thread from above work for you?  please provide as much information about your system and install method as you can.

**Comment by Erwane** [comment]
Got the same on the snap version. Clearing font-cache do nothing.

My log:

```
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (fr_FR.UTF-8)
/bin/bash: /home/erwane/.bashrc: Permission denied
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (fr_FR.UTF-8)
/bin/bash: /home/erwane/.bashrc: Permission denied
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (fr_FR.UTF-8)
/bin/bash: /home/erwane/.bashrc: Permission denied
/bin/bash: /home/erwane/.bashrc: Permission denied
14:30:10.998 › Running version 2022.2.1
/bin/bash: /home/erwane/.bashrc: Permission denied
Error org.freedesktop.DBus.Error.Failed: cannot find desktop file "/var/lib/snapd/desktop/applications/insomnia_Insomnia.desktop"
14:30:11.010 › [electron client protocol] FAILED to set default protocol 'insomnia://'
14:30:11.023 › [electron client protocol] the current executable is not the default protocol for 'insomnia://'
14:30:11.027 › [electron client protocol] the default application set for 'insomnia://' was not found

(insomnia:55163): Gtk-WARNING **: 14:30:11.044: Theme parsing error: gtk.css:1418:23: 'font-feature-settings' is not a valid property name

(insomnia:55163): Gtk-WARNING **: 14:30:11.047: Theme parsing error: gtk.css:3295:25: 'font-feature-settings' is not a valid property name

(insomnia:55163): Gtk-WARNING **: 14:30:11.048: Theme parsing error: gtk.css:3757:23: 'font-feature-settings' is not a valid property name
14:30:11.162 › [fix] Running database repairs
[55163:0428/143011.164148:WARNING:bluez_dbus_manager.cc(248)] Floss manager not present, cannot set Floss enable/disable.
14:30:11.196 › [db] Initialized DB at /home/erwane/snap/insomnia/176/.config/Insomnia/insomnia.$TYPE.db
14:30:11.197 › [db] Init responses DB
14:30:11.200 › [localstorage] Initialized at /home/erwane/snap/insomnia/176/.config/Insomnia/localStorage

(insomnia:55163): dbind-WARNING **: 14:30:11.223: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-RHcG3geuRB: No such file or directory
14:30:11.229 › [main] Loading file:///snap/insomnia/176/resources/app.asar/renderer.html
14:30:11.750 › [updater] Updater not running platform=linux dev=false
[55163:0428/143012.582722:INFO:CONSOLE(33)] "The vm module of Node.js is deprecated in the renderer process and will be removed.", source: node:electron/js2c/renderer_init (33)
[55163:0428/143012.684797:INFO:CONSOLE(33)] "The vm module of Node.js is deprecated in the renderer process and will be removed.", source: node:electron/js2c/renderer_init (33)
[55163:0428/143013.690331:INFO:CONSOLE(0)] "14:30:13.355 › [db] Initialized DB client", source:  (0)
(node:55282) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `exe --trace-deprecation ...` to show where the warning was created)
[55163:0428/143013.828670:INFO:CONSOLE(0)] "14:30:13.418 › The vm module of Node.js is deprecated in the renderer process and will be removed.", source:  (0)
[55163:0428/143013.828723:INFO:CONSOLE(0)] "14:30:13.429 › The vm module of Node.js is deprecated in the renderer process and will be removed.", source:  (0)
[55163:0428/143014.113710:INFO:CONSOLE(0)] "14:30:14.074 › [FileSystemDriver] Initialized in "/home/erwane/snap/insomnia/176/.config/Insomnia/version-control"", source:  (0)
[55163:0428/143014.113765:INFO:CONSOLE(0)] "14:30:14.076 › [FileSystemDriver] Initialized in "/home/erwane/snap/insomnia/176/.config/Insomnia/version-control"", source:  (0)
[55163:0428/143014.113795:INFO:CONSOLE(0)] "14:30:14.076 › [sync] checking for collections which need to be moved into a remote project", source:  (0)
Gtk-Message: 14:30:26.800: GtkDialog mapped without a transient parent. This is discouraged.
Gtk-Message: 14:30:26.854: GtkDialog mapped without a transient parent. This is discouraged.
[55163:0428/143027.287309:INFO:CONSOLE(0)] "14:30:27.277 › [toast] Received notification open-source", source:  (0)
[55163:0428/143037.788858:INFO:CONSOLE(0)] "14:30:37.786 › [toast] Received notification open-source", source:  (0)
[55163:0428/143037.789543:INFO:CONSOLE(0)] "14:30:37.788 › [toast] Not showing notification open-source because has already been seen", source:  (0)
Gtk-Message: 14:30:42.708: GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by filfreire** [comment]
Hi @hadisfr @Erwane @supanadit are you folks still able to reproduce this issue in our [latest stable release](https://github.com/Kong/insomnia/releases/tag/core%402022.4.2)?

**Comment by hadisfr** [comment]
> Hi @hadisfr @Erwane @supanadit are you folks still able to reproduce this issue in our [latest stable release](https://github.com/Kong/insomnia/releases/tag/core%402022.4.2)?

Hi!
Recently, I haven't encountered the aforementioned issue.

**Comment by mieseprem** [comment]
Hi @filfreire ,

I installed Insomnia several minutes ago and faced this issue on about page and file chooser (import data):
![grafik](https://user-images.githubusercontent.com/50367822/180806530-5cdfe4d2-313e-4f6d-9a7d-e2e07487c27b.png)
![grafik](https://user-images.githubusercontent.com/50367822/180806063-8aab64f7-7fa3-4d42-8b24-ccdcf07f187b.png)
However, the preferences page is looking fine:
![grafik](https://user-images.githubusercontent.com/50367822/180806227-a2162098-e823-4dbf-82fb-e1a7aa351925.png)

> [vagrant@devbox ~]$ snap list
> Name               Version                     Rev    Tracking       Publisher    Notes
> bare               1.0                         5      latest/stable  canonical✓   base
> core18             20220706                    2538   latest/stable  canonical✓   base
> gnome-3-28-1804    3.28.0-19-g98f9e67.98f9e67  161    latest/stable  canonical✓   -
> gtk-common-themes  0.1-81-g442e511             1535   latest/stable  canonical✓   -
> insomnia           2022.4.2                    180    latest/stable  getinsomnia  -
> snapd              2.56.2                      16292  latest/stable  canonical✓   snapd

(this is on Fedora 36 workstation)

I already executed these commands (and restarted my session):
> sudo rm -f /var/cache/fontconfig/*
> rm -f ~/.cache/fontconfig/*
> sudo fc-cache -r -v
> find ~/snap/ -wholename '*/.cache/fontconfig'
> rm -rf /home/vagrant/snap/insomnia/common/.cache/fontconfig

**Comment by filfreire** [comment]
@mieseprem thank you! We'll try to investigate this asap.

**Comment by GoNzCiD** [comment]
I have the same issue on OpenSUSE Leap 15.4

**Comment by danfercf1** [comment]
Same issue on import and about screens

```
Version: Insomnia 2022.6.0
Build date: 9/26/2022
OS: Linux x64 5.15.72-1-MANJARO
Electron: 19.0.3
Node: 16.14.2
V8: 10.2.154.4-electron.0
Architecture: x64
```

The logs:

```
[21408:1012/155158.979637:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 5783, .bad_value = 94371870, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155158.979716:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 5785, .bad_value = 85983263, .minor_opcode = 6, .major_opcode = 130}
Gtk-Message: 15:52:00.649: GtkDialog mapped without a transient parent. This is discouraged.
[21408:1012/155200.686960:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 5873, .bad_value = 94371876, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155200.703319:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6013, .bad_value = 85983264, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.759308:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6122, .bad_value = 88082182, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.759441:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6124, .bad_value = 88082181, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.763596:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6126, .bad_value = 11139293, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.765374:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6128, .bad_value = 11139286, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.765548:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6130, .bad_value = 11139287, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.765757:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6132, .bad_value = 11139294, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.765878:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6134, .bad_value = 11139288, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.765940:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6136, .bad_value = 11139289, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.766069:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6138, .bad_value = 11139290, .minor_opcode = 6, .major_opcode = 130}
[21408:1012/155201.766110:WARNING:connection.cc(41)] X error received.  Request: Shape::SelectInputRequest, Error: WindowError{.sequence = 6140, .bad_value = 11139291, .minor_opcode = 6, .major_opcode = 130}
```

**Comment by filfreire** [comment]
Closing as stale. We have updated through a handful of Electron versions since the issue was reported. Please report a new issue if you still face similar problem.

**Comment by srmncnk** [comment]
For anybody experiencing this with snap - the delete fontcache fix worked for me, but only after I deleted the correct path.
For me that was /home/simon/snap/insomnia-designer/common/.cache/fontconfig

After that, it took a few seconds for UI to be loaded, but it worked.

**Comment by srmncnk** [comment]
Funny enough, it breaks right back though.

**Comment by jrlucier** [comment]
This combo fixed it for me.  Make sure you close the app first.

```
sudo rm /var/cache/fontconfig/*
rm ~/.cache/fontconfig/*
fc-cache -r
sudo fc-cache --really-force
```
